ftw/news/portlets/templates/news_portlet.pt

```diff
@@ -0,0 +1,66 @@
+<html xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      tal:omit-tag="python: 1"
+      i18n:domain="ftw.news">
+
+    <dl class="portlet portletCollection newsTemplate"
+        tal:define="items view/get_items;">
+
+        <dt class="portletHeader">
+            <tal:title replace="view/data/news_listing_config_title" />
+        </dt>
+
+        <dd>
+            <ul>
+                <li class="portletItem noRecentNews"
+                    tal:condition="not: items"
+                    i18n:translate="no_recent_news_label">No recent news available.</li>
+
+                <tal:loop tal:repeat="item items">
+                    <li class="portletItem"
+                        tal:define="oddrow repeat/item/odd;
+                                    lastrow repeat/item/last"
+                        tal:attributes="class python:'%s%s' % (
+                                        lastrow and 'lastItem ' or '',
+                                        oddrow and 'portletItem even' or 'portletItem odd')">
+
+                        <div class="newsText">
+                            <div class="portletItemDate"
+                                 tal:content="item/news_date" />
+
+                            <div class="portletItemTitle" tal:content="item/title" />
+
+                            <div class="portletItemDescription"
+                                 tal:condition="item/description">
+                                <div tal:content="item/description" />
+                            </div>
+                        </div>
+
+                        <a class="portletItemMore"
+                           tal:attributes="href item/url;"
+                           i18n:translate="label_readmore">Read more</a>
+
+                        <div class="visualClear"><!-- --></div>
+                    </li>
+                </tal:loop>
+
+                <li class="portletFooter"
+                    tal:define="more_news_url view/more_news_url;
+                                show_rss_link view/show_rss_link">
+                    <tal:morelink tal:condition="more_news_url">
+                        <a class="newsMoreLink" title="More News"
```

Comment:
`title` is [not translated](https://jenkins.4teamwork.ch/view/All/job/ftw.news-pull-request-1-test-plone-4.3.x.cfg/Translation_Report/)